mantle/platform/machine/aws/cluster.go

```diff
@@ -57,7 +57,7 @@ func (ac *cluster) NewMachineWithOptions(userdata *conf.UserData, options platfo
 		return nil, errors.New("platform aws does not support appending firstboot kernel arguments")
 	}
 
-	conf, err := ac.RenderUserData(userdata, map[string]string{
+	conf, err := ac.BaseCluster.RenderUserData(userdata, map[string]string{
```

Comment:
Hmm, there are a lot of these changes.  This essentially adds an implementation detail into the invocation, which is kind of ugly.  Why does the linter complain about it?

Reply:
That's... weird. I swear initially when I ran this locally, it was triggering a bunch of lint errors. But reverting now I can't seem to reproduce it anymore.

Anyway, dropped those!